administration/src/project-configs/bayern/deepLinkConfig.ts

```diff
@@ -0,0 +1,17 @@
+import { APP_STORE_TEAM_ID, buildConfigBayern } from 'build-configs'
```

Comment:
i think i didn't get your idea how to accomplish this.
How should i make two config files with different content generic?
I mean i could add the content of these configs to `frontend/build-configs` and then pass them.

Reply:
i'm not sure if i got the whole picture of how to accomplish what you want.

I mean we need a different config for each project because the content is different for every whitelabel. But I moved this config now to `fronted/buildConfigs`
Not sure if it makes sense to pass the particular `buildConfig` in webpack, since we only have one build not like integreat. So i execute the `getDeepLinkingConfigs()` where i have listed the particular build-configs in an array for all build-configs where we need the association files. Feel free to add a proposal or an example of what you would recommend if that is not fine for you, @ztefanie